_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,89 @@
+---
+title: New API Blueprint tutorial
```

Comment:
Revise and modify. Title implies an actual tutorial, but the body of the content is a description of how the tutorial was revamped and simplified.